<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 255, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>951</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 58);</string>
    </property>
    <widget class="QPushButton" name="buttonDersEkle">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(137, 255, 145);</string>
     </property>
     <property name="text">
      <string>Ders Ekle</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableDerslerWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>931</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>152</number>
     </attribute>
     <column>
      <property name="text">
       <string>Ders Adi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ders Günü</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ders Saati</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ders Süresi </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ders Linki</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(133, 233, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>45</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Kullanıcı Adı:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Şifre:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textAlKullaniciAd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textAlKullaniciSifre">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBilgiGuncelle">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>122</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Güncelle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonProgramDurumDegistir">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Programı Başlat</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDurumYaz">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
